plugins/module_utils/copy.py

```diff
@@ -108,12 +121,66 @@ def copy_pds2uss(src, dest):
     src = _validate_data_set_name(src)
     dest = _validate_path(dest)
     cp_pds2uss = "cp -U -F rec \"//'{0}'\" {1}".format(src, quote(dest))
+    if is_binary:
+        cp_pds2uss = cp_pds2uss.replace("rec", "bin")
     rc, out, err = module.run_command(cp_pds2uss)
     if rc:
         raise USSCmdExecError(cp_pds2uss, rc, out, err)
     return rc, out, err
 
 
+def copy_uss2uss_binary(src, dest):
+    """Copy a USS file to a USS location in binary mode
+
+    Arguments:
+        src: {str} -- The source USS path
+        dest: {str} -- The destination USS path
+    Raises:
+        USSCmdExecError: When any exception is raised during the conversion.
+    Returns:
+        boolean -- The return code after the USS command executed successfully
+        str -- The stdout after the USS command executed successfully
+        str -- The stderr after the USS command executed successfully
+    """
+    module = AnsibleModule(argument_spec={}, check_invalid_arguments=False)
+    src = _validate_path(src)
+    dest = _validate_path(dest)
+    cp_uss2uss = "cp -F bin {0} {1}".format(quote(src), quote(dest))
+    rc, out, err = module.run_command(cp_uss2uss)
+    if rc:
+        raise USSCmdExecError(cp_uss2uss, rc, out, err)
+    return rc, out, err
+
+
+def copy_mvs2mvs(src, dest, is_binary=False):
+    """Copy an MVS source to MVS target
+
+    Arguments:
+        src: {str} -- Name of source data set
+        dest: {str} -- Name of destination data set
+
+    Keyword Arguments:
+        is_binary: {bool} -- Whether the data set to be copied contains binary data
+
+    Raises:
+        USSCmdExecError: When any exception is raised during the conversion.
+    Returns:
+        boolean -- The return code after the USS command executed successfully
+        str -- The stdout after the USS command executed successfully
+        str -- The stderr after the USS command executed successfully
+    """
+    module = AnsibleModule(argument_spec={}, check_invalid_arguments=False)
+    src = _validate_data_set_name(src)
+    dest = _validate_data_set_name(dest)
+    cp_mvs2mvs = "cp -F rec \"//'{0}'\" \"//'{1}'\"".format(src, dest)
+    if is_binary:
```

Comment:
Since Python's string replacement method `replace()` replaces all occurrences by default, I think this may result in unexpected behavior when the *src* or *dest* contain the string "rec". Would it be possible do to one of the following, or something different if you prefer:

1. Add the count argument of `1` to each call to `replace()`, probably the easiest change: [more info here](https://docs.python.org/3/library/stdtypes.html#str.replace)
2. update the strings to take another argument to determine if `rec` or `bin` should be used, basic example:
```python
copy_type = "bin" if is_binary else "rec"
cp_uss2mvs = "cp -CM -F {0} {1} \"//'{2}'\"".format(copy_type, quote(src), dest)
```
I personally find second method a bit cleaner but that is just personal preference